workshops/konami_code/README.md

```diff
@@ -0,0 +1,478 @@
+---
+name: 'Konami Code'
+description: 'Implement a konami code on a website'
+author: '@faisalsayed10'
+---
+
+# Konami Code
+
+You may have heard about movie Easter Eggs or video game Easter Eggs before, but have you ever heard about website Easter Eggs? The internet is filled with copious amounts of weird and wonderful hidden features! I bet you have played the popular game GTA, right? GTA too had konami codes but they were called 'cheat codes'. 
+
+If you don't know what they are, check out these cool easter eggs/konami codes on the internet:
+
+1. Try searching 'do a barrel roll' on google.
+
+2. Try inspecting [Max's Website](maxwofford.com) 🤨
+
+3. Konami Code On [BuzzFeed](https://www.buzzfeed.com/) (Just press up, up, down, down, left, right, left, right, B, A and enter on the homepage).
+
+4. In the Firefox browser bar, type "about:mozilla" or "about:robots" to see the secrets of Mozilla.
+
+Well, by now, you must have known what a konami code is. How about we build our own konami code?
+
+It will look something like this after someone types the konami code:
+
+[![Image](https://cloud-5apj3kt1e.vercel.app/image.png)](https://konami-code.faisalsayed1.repl.co/)
+
+Here's the [live demo](https://konami-code.faisalsayed1.repl.co/)  
+Here's the [source code](https://repl.it/@FaisalSayed1/Konami-Code)
+
+Live demo: 
+![Image](https://cloud-95l2730mf.vercel.app/konamigif.gif)
+
```

Comment:
```suggestion

It will look something like this after someone types the Konami code:

[![Image](https://cloud-5apj3kt1e.vercel.app/image.png)](https://konami-code.faisalsayed1.repl.co/)

Here's the [live demo](https://konami-code.faisalsayed1.repl.co/)  
Here's the [source code](https://repl.it/@FaisalSayed1/Konami-Code)

Live demo: 
![Image](https://cloud-95l2730mf.vercel.app/konamigif.gif)

```

Reply:
What are the changes here? there is nothing different? why then? a review?